spec/helpers/timestamper.rb

```diff
@@ -0,0 +1,31 @@
+require 'time_difference'
+
+module Timestamper
+
+  def relative_time(creation_time)
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for relative_time is too high. [33.26/15]